### ISSUE TITLE
WCAG 3.1.2 ✅ (Supports): Language of Parts

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#5158]
**Level:** AA
**Principle:** Understandable
**Result:** ✅ Supports

### Resources
[WCAG 2.2 Success Criterion](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html)

> The [human language](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html#dfn-human-language) of each passage or phrase in the content can be [programmatically determined](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html#dfn-programmatically-determined) except for proper names, technical terms, words of indeterminate language, and words or phrases that have become part of the vernacular of the immediately surrounding [text](https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts.html#dfn-text).

### Audit Checklist
- [x] Review previous Audit Result
- [x] Review work identified in previous audit and list open issues.
- [x] Check for any other relevant open issues.
- [x] Retest open issues, are these issues still present?
- [x] Test for any new issues that may have been introduced.
- [ ] Determine compliance status, add this to issue name.
- [ ] Add any new issues to the Accessibility project for triage.

### Previous audit
> ✅ language of parts
All standard parts of the page are in the language of the page. Any part that differs from the language of the page would be due to the user-content on that page. While this is user content, we are looking at ways to better support the markup of such parts.

- https://github.com/openlibhums/janeway/issues/4218

 ### Progress since last audit

There has been no change since the last audit. Work on supporting users specifying language of parts is set for a later phase of the accessibility project so the issue remains open.

 ### Audit Notes

Text on the Front of house that is part of the codebase is single language per page, and so there is no failure of language of parts. 

User content may include phrases in other languages.  If this is on the static pages - then this can be marked up in HTML in the editor.  For articles, if the typesetter has marked up the text with a "lang" attribute, this will be carried across directly into the HTML markup, so that the language of that part is set. 

In conclusion, we support language of parts, but it is reliant on correct markup of user content. There is none of our own code-base text that requires language of parts markup. 

 ### Work arising

There are currently 3 open issues relating to this SC.  They all relate to better support for user content, rather than bugs in compliance. 
- #4139
- #4187
- #4218
